### PR TITLE
added js comments to NavBar.js

### PR DESCRIPTION
**Pull request by nicholasvaldez**
# Description

added js comments to NavBar.js

# Steps to test

1. fetch and checkout the branch nick-nav
2. open NavBar.js 
3. Verify comments are accurately describing boilerplate code

### CODE REVIEW

File: src/components/nav/NavBar.js [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+// uses Props in the argument defined in Rare.js
 export const NavBar = ({ token, setToken }) => {
-  const navigate = useNavigate()
-  const navbar = useRef()
-  const hamburger = useRef()
+  //defines navigate variable to use useNavigate hook
+  const navigate = useNavigate();
 
+  /*
+  •useRefs() hook are used for functionality of the mobile nav bar, which appears when viewport width is decreased
```
Comment (MaeYoungPhan) [inline src/components/nav/NavBar.js:13]: Please clarify this comment about useRefs()

File: src/components/nav/NavBar.js [hunk 2]
```diff
@@ -30,35 +58,42 @@ export const NavBar = ({ token, setToken }) => {
...
       <div className="navbar-menu" ref={navbar}>
         <div className="navbar-start">
-          {
-            token
-              ?
-              <Link to="/" className="navbar-item">Posts</Link>
-              :
-              ""
-          }
+          {token ? (
```
Comment (MaeYoungPhan) [inline src/components/nav/NavBar.js:61]: Please add a comment explaining what this ternary statement is doing

File: src/components/nav/NavBar.js [hunk 3]
```diff
@@ -30,35 +58,42 @@ export const NavBar = ({ token, setToken }) => {
...
-                    setToken('')
-                    navigate('/login')
-                  }}>Logout</button>
-                  :
-                  <>
-                    <Link to="/register" className="button is-link">Register</Link>
-                    <Link to="/login" className="button is-outlined">Login</Link>
-                  </>
-              }
+              {token ? (
```
Comment (MaeYoungPhan) [inline src/components/nav/NavBar.js:73]: Please leave a comment explaining this code

**Review by MaeYoungPhan — COMMENTED** [review]
See comments in code. Almost there! Great work!

File: src/components/nav/NavBar.js (hunk 1 above)
Comment (nicholasvaldez) [reply src/components/nav/NavBar.js]: done 👍

File: src/components/nav/NavBar.js (hunk 2 above)
Comment (nicholasvaldez) [reply src/components/nav/NavBar.js]: done 👍

File: src/components/nav/NavBar.js (hunk 3 above)
Comment (nicholasvaldez) [reply src/components/nav/NavBar.js]: done 👍

**Pull request merged**